src/sonic-config-engine/tests/test_cfggen.py

```diff
@@ -49,6 +59,32 @@ def tearDown(self):
 
     def run_script(self, argument, check_stderr=False, verbose=False):
         print('\n    Running sonic-cfggen ' + argument)
+        if PY3x and "-m" in argument:
+            pattern = r'-m\s+(\S+)\s*'
+            minigraph = re.findall(r'-m\s+(\S+)\s*', argument)
+            hwsku     = re.findall(r'-S\s+(\S+)\s*', argument)
+            port_conf = re.findall(r'-p\s+(\S+)\s*', argument)
+            namespace = re.findall(r'-n\s+(\S+)\s*', argument)
+            if minigraph:
+                print('\n    Validating yang schema')
+                cmd = self.script_file + ' -m ' + minigraph[0]
+                if hwsku:
+                    cmd += ' -S ' + hwsku[0]
+                if port_conf:
+                    cmd += ' -p ' + port_conf[0]
+                if namespace:
+                    cmd += ' -n ' + namespace[0]
+                cmd += ' --print-data'
+                output = subprocess.check_output(cmd, shell=True).decode()
+                # "NULL": "NULL" is placeholder for redis, remove to pass yang validation.
```

Comment:
>"NULL": "NULL" [](http://example.com/codeflow?start=18&length=14)

This seems a bug. I submit a PR to fix [https://github.com/Azure/sonic-buildimage/pull/9703](https://github.com/Azure/sonic-buildimage/pull/9703).